instaclustr/data_source_clusters.go

```diff
@@ -0,0 +1,85 @@
+package instaclustr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceClustersList() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceClustersListRead,
+
+		Schema: map[string]*schema.Schema{
+			"cluster": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cluster_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"cluster_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"cassandra_version": {
```

Comment:
Do you have a current need for this field? Else for forward compatibility purposes for when we decide to properly hook in the right APIs and schema to this data source, could we remove `cassandra_version` ?

Reply:
Removed this field